t/t.go

```diff
@@ -168,6 +168,7 @@ func outputLogs(prefix string) {
 		out, err := logCmd.CombinedOutput()
 		x.Check(err)
 		f.Write(out)
+		fmt.Println(string(out))
 		// fmt.Printf("Docker logs for %s is %s with error %+v ", c.ID, string(out), err)
```

Comment:
open line #172, and rm line#171

Reply:
pls verify the CI run logs to make sure you have the right log statements

Reply:
will have to wait for a failure, 172 opened